Assets/Editor/Dialogue Editor/DialogueGraphView.cs

```diff
@@ -97,14 +97,22 @@ public DialogueNode CreateDialogueNode(string nodeName) {
 
     //adds choices to node
         //output ports are invisible for some reason
-    public void AddChoicePort(DialogueNode dialogueNode, string overriddenPortName = "") {
+    public void AddChoicePort(DialogueNode dialogueNode, string overriddenPortName = "", string defaultPortName = null) {
         var generatePort = GeneratePort(dialogueNode,Direction.Output);
        
         var oldLabel = generatePort.contentContainer.Q<Label>("type");
         generatePort.contentContainer.Remove(oldLabel);
 
         var outputPortCount = dialogueNode.outputContainer.Query("connector").ToList().Count;
-        var choicePortName = string.IsNullOrEmpty(overriddenPortName) ? $"Choice {outputPortCount}" : overriddenPortName;
+        string choicePortName;
+        if (defaultPortName == null)
+        {
+            choicePortName = string.IsNullOrEmpty(overriddenPortName) ? $"Choice {outputPortCount}" : overriddenPortName;
+        }
+        else
+        {
+            choicePortName = string.IsNullOrEmpty(overriddenPortName) ? defaultPortName : overriddenPortName;
+        }
```

Comment:
a cleaner way to do this would be to set defaultPortName initially to empty in the method header (since you're saying empty and null mean the same thing here), and then just use the second expression, like 
```cs
defaultPortName ??= $"Choice {outputPortCount}"
choicePortName = string.IsNullOrEmpty(overriddenPortName) ? defaultPortName : overriddenPortName;
```
the ??= operator replaces the value iff the left side of the expression is null